stylesheets/_component.panels.scss

```diff
@@ -19,6 +19,7 @@
 
     &__body {
         max-height: 20em;
+        overflow-y: auto;
```

Comment:
There's some additional work required to make scrolling containers keyboard accessible. See: https://developer.paciellogroup.com/blog/2016/02/short-note-on-improving-usability-of-scrollable-regions/

Note: this is subjective depending on if the scrolling container has focusable elements within it.